statsmodels/nonparametric/kernel_regression.py

```diff
@@ -875,15 +875,15 @@ def _compute_test_stat(self, Y, X):
 
         m0 = model.fit(data_predict=X1)[0]
         m0 = np.reshape(m0, (n, 1))
-        I = np.zeros((n, 1))
+        I_n = np.zeros((n, 1))
```

Comment:
Does `I` have a domain-specific meaning here I'm not aware of?  The other cases are identity matrices.  Unless this has a specific meaning, I might call this `zvec`